### PR TITLE
Led class and thread created

### PR DESCRIPTION
**Pull request by stoneoxmike**
Not implemented yet, but the framework is there. Should only be enabled for the Orin, and the legacy implementation should work for the Pi

### CODE REVIEW

File: src/classes/LEDClass.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class LEDClass:
```
Comment (Mitchell-Conrad-20) [inline src/classes/LEDClass.py:1]: Can we name this better? the class name shouldnt have "class" in it

File: src/config/global_vars.py [hunk 2]
```diff
@@ -246,16 +246,12 @@ def init():
...
-        24,                             # Number of LEDs (Num of Pixels)
-        brightness = 0.05,              # Scale from 0.00 to 1.00 (Higher = Brighter), CAUTION: 1.00 hurts your eyes
-        pixel_order = neopixel.GRB      # G and R are reversed, so the colors are actually in order of RGB
-    )
-    
-    
-    
-
+    pixels = None
+    if robot == "pi":
```
Comment (Mitchell-Conrad-20) [inline src/config/global_vars.py:251]: lets choose one spot to setup neopixel for spi or regular. do it in the same spot, not two diff files. id prefer to do it in globals tbh

**Review by Mitchell-Conrad-20 — CHANGES_REQUESTED** [review]
See comments on files

**Comment by stoneoxmike** [comment]
Merged to Legacy-Mode-Changes

**Pull request closed without merging**